RealmSwift-swift2.0/Realm.swift

```diff
@@ -279,8 +279,7 @@ public final class Realm {
     - returns: The created object.
     */
     public func create<T: Object>(type: T.Type, value: AnyObject = [:], update: Bool = false) -> T {
-        // FIXME: use T.className()
-        let className = (T.self as Object.Type).className()
+        let className = (type as Object.Type).className()
```

Comment:
The fixme was there because I honestly don't understand why `type.className()` doesn't yield the same result as `(type as Object.Type).className()`.